packages/reactotron-app/App/Commands/StateActionCompleteCommand.js

```diff
@@ -1,29 +1,27 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import Command from '../Shared/Command'
-import Colors from '../Theme/Colors'
-import Content from '../Shared/Content'
+import { observer } from "mobx-react"
+import PropTypes from "prop-types"
+import React, { Component } from "react"
+import Command from "../Shared/Command"
+import Content from "../Shared/Content"
+import Colors from "../Theme/Colors"
 
-const COMMAND_TITLE = 'ACTION'
+const COMMAND_TITLE = "ACTION"
 
 const Styles = {
   name: {
     color: Colors.bold,
     margin: 0,
-    paddingBottom: 10
-  }
+    paddingBottom: 10,
+  },
 }
 
+@observer
```

Comment:
Do we need to make all the commands observable? Is this because of the MobX upgrade?

Reply:
I'm not sure we do.  I think the answer to "will it ever change?" is no.